relm-core/examples/main.rs

```diff
@@ -47,11 +47,18 @@ use relm_core::EventStream;
 
 use Msg::*;
 
+
+// There will be several widgets involved in this example, but this struct
+// will act as a container just for the widgets that we will be updating
 struct Widgets {
     clock_label: Label,
     counter_label: Label,
 }
 
+// This enum holds the various messages that will be passed between our
+// widgets. Note that we aren't deriving `Msg` because this is example
+// is barebones and isn't meant to instruct you how to create a typical
```

Comment:
This doesn't use `Msg` because it only uses the stuff from `relm-core` which is the basic library on which `relm` is built.